src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/action/SearchJQLSubset.kt

```diff
@@ -0,0 +1,38 @@
+package com.atlassian.performance.tools.jiraactions.api.action
+
+import com.atlassian.performance.tools.jiraactions.api.SEARCH_JQL_SUBSET
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import com.atlassian.performance.tools.jiraactions.api.memories.IssueKeyMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.JqlMemory
+import com.atlassian.performance.tools.jiraactions.api.observation.SearchJqlObservation
+import com.atlassian.performance.tools.jiraactions.api.page.IssueNavigatorPage
+import com.atlassian.performance.tools.jiraactions.jql.BuiltInJQL
+import java.util.function.Predicate
+import javax.json.JsonObject
+
+class SearchJQLSubset(
```

Comment:
Not that it's very important but the proper camel case would be `SearchJqlSubset`

Reply:
If this will be the only issue on this PR, I will merge and refactor in the next one.